### PR TITLE
Make tty.sh work on a wider variety of systems

### PR DESCRIPTION
**Pull request by badeaffe**

### CODE REVIEW

File: backends/tty/tty.sh [hunk 1]
```diff
@@ -1,20 +1,18 @@
-#!/usr/bin/env bash
+#!/bin/sh
```
Comment (jumper149) [inline backends/tty/tty.sh:1]: To allow other shells this should be:
```
#!/usr/bin/env sh
```

File: backends/tty/tty.sh [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+printf "\033]P${7}"
+printf "\033]P${8}"
+printf "\033]P${9}"
+printf "\033]P${10}"
+printf "\033]P${11}"
+printf "\033]P${12}"
+printf "\033]P${13}"
+printf "\033]P${14}"
+printf "\033]P${15}"
+printf "\033]P${16}"
```
Comment (jumper149) [inline backends/tty/tty.sh:18]: This is good.

File: backends/tty/tty.sh (hunk 1 above)
Comment (badeaffe) [reply backends/tty/tty.sh:1]: However, `/bin/sh` is usually a symlink to another executable anyway.

Comment (jumper149) [reply backends/tty/tty.sh:1]: Hmm you're actually right. `/bin/sh` is fine :)

**Pull request merged**